mods/noita-mp/files/scripts/Ui.lua

```diff
@@ -236,13 +236,94 @@ function Ui.new()
 
         menuWidth, menuHeight = renderEzgui(x, y, "mods/noita-mp/files/data/ezgui/PlayerList.xml", self.ezguiMenuData)
     end
-
     ------------------------------------
     -- Public methods:
     ------------------------------------
+    local missingModGuiButton1Hovered = false
+    local missingModGuiButton2Hovered = false
+    local missingModGuiDismissed = false
     function self.update()
         drawFolding()
         drawMenu()
+        if _G.whoAmI() == Client.iAm and Client.missingMods ~= nil and not missingModGuiDismissed then
```

Comment:
Would be awesome to put this in a 'private' local function like 'drawMenu()'. It will be easier for me, when reworking UI stuff.